gcloud/bigtable/row.py

```diff
@@ -253,3 +324,21 @@ def to_pb(self):
         :returns: The converted current object.
         """
         return data_pb2.RowFilter(cells_per_column_limit_filter=self.num_cells)
+
+
+class StripValueTransformerFilter(_BoolFilter):
```

Comment:
On it. I debated doing that but figured putting them at the bottom might make them easier to review. Will ping once done.